### DISCUSSION TITLE
PNG not rendered

### DISCUSSION THREAD

**Discussion by artincube** [photoprism/photoprism#4353] [Q&A] [answered]
I have a bunch of very large png (some over 12500x12500 16 bits) with metadata.
They import fine including metadata (GPS included) after I remove the resolution limit; thumbnails are rendered, but no scaled down jpeg in sidecar, only the ymls.
So the full size png (around 500MB each, largest 1.32GB) is displayed in browser when showing the images, it seems.
Not very efficient.
Why is Photoprism not rendering these previews? Did I miss something?

**Comment by lastzero** [comment]
I just tested this with a regular (not extremely large) PNG file and the viewer displayed a JPEG instead of the original PNG file.

As mentioned in our docs, if the configured size limits under Settings > Advanced are exceeded and you have a large screen, a sufficiently large preview image cannot be created and the photo viewer may be forced to display the original image instead:
- https://docs.photoprism.app/user-guide/settings/advanced/#static-and-dynamic-size-limits
- https://docs.photoprism.app/user-guide/settings/advanced/#jpeg-size-limit

Unfortunately, you didn't provide complete information regarding your settings, so we can only guess what happened.

**Comment by artincube** [comment]
I think that is a very good guess from the information I gave you. Indeed if I make the browser window smaller then it loads the largest thumbnail very quickly. My limits at 2560 are easily exceeded on the MacBook Pro laptop.But I would rather have the largest thumbnail resized bigger and pixelated by the browser instead of a 12000 pixel 16 bits file shrunken.I still don’t fully understand why the large png don’t get a preview created in sidecar as a tiff or other format of the same size. I get that png  are supposed to be compatible with all browsers, but what if size is too big for that purpose. On Jun 23, 2024, at 11:04 PM, Michael Mayer ***@***.***> wrote:﻿
I just tested this with a regular (not extremely large) PNG file and the viewer displayed a JPEG instead of the original PNG file.
As mentioned in our docs, if the configured size limits under Settings > Advanced are exceeded and you have a large screen, a sufficiently large preview image cannot be created and the photo viewer may be forced to display the original image instead:

https://docs.photoprism.app/user-guide/settings/advanced/#static-and-dynamic-size-limits
https://docs.photoprism.app/user-guide/settings/advanced/#jpeg-size-limit

Unfortunately, you didn't provide complete information regarding your settings, so we can only guess what happened.

**Reply by lastzero** [reply]
Try to set the dynamic size limit to the maximum value and allow dynamic rendering of thumbnails under Settings > Advanced.

**Reply by artincube** [reply, marked as answer]
That is the solution. I enabled dynamic and big size. The previews are rendered as smaller jpeg before being displayed.

The dynamically rendered jpegs are saved in thumbnails I assume as after the first view they load fast jpegs (no rendering I think) even after emptying caches in safari.

Thank you!!!
